src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  background-color: black;
+  margin: 0;
+}
+
+#qa-block {
+  margin-top: 20px;
+}
```

Comment:
IDs are used for JS, try not to style the id if you can avoid it. This makes impossible to reuse the code. Stylize classes. [Style guide](https://mate-academy.github.io/style-guides/htmlcss.html#id-selectors)